### DISCUSSION TITLE
Website down?

### DISCUSSION THREAD

**Discussion by jacksorjacksor** [katspaugh/wavesurfer.js#3816] [Q&A]
Hi there!

https://wavesurfer.xyz/ appears to be down - is that a known issue?

Cheers!

Rich

**Comment by weavermedia** [comment]
It's working OK for me in Los Angeles.

**Comment by katspaugh** [comment]
Works for me as well, CloudFlare status is green too: https://www.cloudflarestatus.com

**Comment by jacksorjacksor** [comment]
Weird! I'm still getting the same errors as yesterday. I'm based in the UK FWIW.

Using Edge gets me:

"It looks like wavesurfer.xyz closed the connection"

![image](https://github.com/user-attachments/assets/6d38fe11-be3b-4621-9ef1-60f42eedc2e1)

Using Firefox gets me:

"Secure Connection Failed - Error code: PR_END_OF_FILE_ERROR"

![image](https://github.com/user-attachments/assets/50c36624-0ebf-454c-a092-39e261814725)

Any advice would be appreciated!

TIA!

Rich
